### PR TITLE
fix(bazel): prod server doesn't serve files in windows 

### PR DESCRIPTION
**Pull request by alan-agius4**
In Windows runfiles are not always available thus we need to use the real location using rlocation.

Related to bazelbuild/rules_nodejs#1699

### CODE REVIEW

**Comment by alan-agius4** [comment]
Targeting master only because the fix is also dependent on https://github.com/bazelbuild/rules_nodejs/commit/d635dca which is available only in bazel nodejs rules 1.4.0.

**Review by gregmagolan — APPROVED** [review]
LGTM

File: packages/bazel/src/schematics/ng-add/index.ts [hunk 1]
```diff
@@ -39,14 +39,14 @@ function addDevDependenciesToPackageJson(options: Schema) {
     const devDependencies: [string, string][] = [
       ['@angular/bazel', angularCore.version],
       ['@bazel/bazel', '2.1.0'],
-      ['@bazel/ibazel', '0.11.1'],
+      ['@bazel/ibazel', '0.12.2'],
```
Comment (gregmagolan) [inline packages/bazel/src/schematics/ng-add/index.ts]: nit: also update in root package.json?

**Review by kyliau — APPROVED** [review]

File: packages/bazel/src/schematics/ng-add/index.ts (hunk 1 above)
Comment (alan-agius4) [reply packages/bazel/src/schematics/ng-add/index.ts]: I’ll do that tmr cause I am current AFK

**Pull request closed without merging**